packages/components/src/components/button-tile/button-tile-utils.ts

```diff
@@ -0,0 +1,9 @@
+import type { Theme } from '../../utils/theme';
+import type { ButtonVariant, LinkButtonIconName } from '../../types';
+
+export type ButtonTileProps = {
```

Comment:
Confusing name. which suggests that is is supposed to be used on the button tile component.